package.json

```diff
@@ -33,6 +34,12 @@
     "@size-limit/preset-small-lib": "^4.10.1",
     "@solidity-parser/parser": "^0.12.0",
     "@types/object-hash": "^2.1.0",
+    "apollo": "^2.33.0",
```

Comment:
`apollo` should be a devDependency. 
![image](https://user-images.githubusercontent.com/74881121/117116519-c232c300-ad8e-11eb-86e8-98a130f77502.png)

Reply:
... and probably the other ones should be directly included in the dependencies, not as peer